picard/acoustid/__init__.py

```diff
@@ -80,10 +80,12 @@ def _on_lookup_finished(self, next_func, file, document, http, error):
                     if results:
                         result = results[0]
                         file.metadata['acoustid_id'] = result['id']
-                        if 'recordings' in result:
+                        if 'recordings' in result and result['recordings']:
+                            max_sources = max([r['sources'] for r in result['recordings']])
                             for recording in result['recordings']:
                                 parsed_recording = parse_recording(recording)
                                 if parsed_recording is not None:
+                                    parsed_recording['score'] = recording['sources'] / max_sources * 100
```

Comment:
Just to be sure, `max_sources` is never zero, right?

Reply:
Good point. Actually it should never be 0, always at least 1. But given that this comes from an external service, and we might get unexpected results, we should probably ensure it is set to 1

Reply:
Yes, better safe than sorry. We had such issue in the past (division by zero exception because the ws wasn't returning what was expected, or just because it was the normal behavior but not taken in account in the code).

Reply:
I made the code more error tolerant, also dealing with the case that this field is not specified at all. We can still fail with a `TypeError` here if `score` would contain something that is not an int, but this at least would be catched by the outer exception handling and Picard would not crash.